ETL-Project/schema/create_administrartion_ecommerce.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS administrators (
+id SERIAL,
+username VARCHAR(45) NOT NULL,
+email VARCHAR(45) NOT NULL,
+password TEXT NOT NULL,
+PRIMARY KEY (id)
+);
```

Comment:
Please add a new line at the end of file
```suggestion
);

```